packages/front-end/components/Queries/ExpandableQuery.tsx

```diff
@@ -93,17 +99,17 @@ const ExpandableQuery: FC<{
             Took{" "}
             {formatDistanceStrict(
               getValidDate(query.startedAt),
-              // @ts-expect-error TS(2345) If you come across this, please fix it!: Argument of type 'Date | undefined' is not assigna... Remove this comment to see the full error message
               getValidDate(query.finishedAt)
             )}
           </em>
         </small>
       )}
-      {query.status === "running" && (
+      {(query.status === "running" || query.status === "queued") && (
         <div className="alert alert-info">
           <em>
-            Running for{" "}
-            {formatDistanceStrict(getValidDate(query.startedAt), new Date())}
+            Running{query.dependencies?.length ? " or queued" : ""}
+            {" for "}
+            {formatDistanceStrict(getValidDate(query.createdAt), new Date())}
```

Comment:
Feels like we should fork the display here and treat queued separately from running.

If the status is queued, we can say:
> Queued for {...}

If the status is running, we can say:
> Running for {...}

For running queries, if we're worried about someone looking 2 minutes in and seeing "Running for 10 seconds" and wondering what it's been doing this whole time, we could take the difference between startedAt/createdAt and show a `Queued for {...}` message.